src/TLuaInterpreter.cpp

```diff
@@ -12033,7 +12094,7 @@ void TLuaInterpreter::setGMCPTable(QString& key, const QString& string_data)
             return;
         }
     }
-    parseJSON(key, string_data, "gmcp");
+    parseJSON(key, string_data, QLatin1String("gmcp"));
```

Comment:
I have trying to compiling Clazy for the Qt Creator/OS I am using right now.

For an ASCII literal string I thought it was okay to use `QLatin1String` for a string that is used in multiple places (because `QStringLiteral` creates a *separate* copy for each instance in the read-only part of the executable)?

Reply:
Maybe, but it's a size vs speed tradeoff - and string literal is faster.